net-news/liferea/liferea-1.12.7.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit gnome2 python-single-r1
+
+DESCRIPTION="News Aggregator for RDF/RSS/CDF/Atom/Echo feeds"
+HOMEPAGE="https://lzone.de/liferea/"
+SRC_URI="https://github.com/lwindolf/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~x86"
+IUSE="gnome-keyring mediaplayer networkmanager notification tray"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+CDEPEND="${PYTHON_DEPS}
```

Comment:
Use `COMMON_DEPEND` instead.

Reply:
Could you please explain if things changed from our previous conversation about CDEPEND?( https://github.com/gentoo/gentoo/pull/10717#discussion_r244667548 ).
As I see the number of ebuilds with it is almost same and 2 eclasses using it also.

Reply:
You can grep the tree for a lot of older code snippets that you wouldn't use today. We try to anticipate and prevent, and not like this is a huge chance in the ebuild (that you'll have to fix anyway). 
Proxy-maintainers don't have commit access, so it's a bit different from people that, if needed, can fix their ebuilds within 30 seconds. 
FYI: I didnt agree on pushing CDEPEND last time, and I still dont. Someone else might.